### ISSUE TITLE
Determining error in peak magnitude

### ISSUE THREAD

**Issue by janeglanzer** [sncosmo/sncosmo#207]
From SNCOSMO, I am able to determine the peak magnitude of the supernova I am working with, but is there a way to calculate the error in the peak magnitude? Tried looking through some of the source code to figure out a way to do this, but couldn't find anything specific on it.

**Comment by kbarbary** [comment]
Yeah it isn't obvious: `fit_lc()` returns the error on the model parameters, including the scale parameter (`x0` or `amplitude` depending on the model), but it is up to the user to translate this to an error on peak magnitude. Here's an example of how to do that:

First, I'll repeat the set up from the [fitting a light curve example](http://sncosmo.readthedocs.io/en/latest/examples/plot_lc_fit.html):

```python
>>> import sncosmo
>>> data = sncosmo.load_example_data()
>>> model = sncosmo.Model(source='salt2')
>>> result, fitted_model = sncosmo.fit_lc(data, model, ['z', 't0', 'x0', 'x1', 'c'], bounds={'z':(0.3, 0.7)})
```

The peak magnitude is
```python
>>> fitted_model.source_peakmag('bessellb', 'vega')
22.907245988703636
```
The fractional error on the scale parameters `x0` is
```python
>>> result.errors['x0'] / result.parameters[2]
0.032705042536128541
```

Translate this to an error in magnitude with `dmag = 2.5 / log(10) * dflux / flux`:
```python
>>> 2.5 / math.log(10) * result.errors['x0'] / result.parameters[2]
0.035509048759629382
```

It would be good to add this to the "fitting a light curve" example in the docs.

**Comment by janeglanzer** [comment]
Thanks for this! Very helpful!

**Comment by kbarbary** [comment]
Great! I'll leave this open as a TO-DO item: adding this info to the docs somewhere.
